src/pages/about.js

```diff
@@ -29,19 +28,22 @@ const AboutPage = ({data}) => (
       <div><b>Faculty Advisor</b>: Kevin Lanke</div>
       
       <h3>Room Info</h3>
-      <p>We have three main rooms in BSB basement: the Broadcasting Room (BSB032), Recording Studio (BSB030), and Podcasting Studio (BSB028).
+      <p>We have four main rooms in BSB basement: the Broadcasting Room (BSB032), Recording Studio (BSB030), Podcasting Studio (BSB028), and Music Lounge (BSB022).
```

Comment:
We probably want to mention the music lounge is actually owned by Res Life/Student Affairs, with us providing equipment, but this works for now